src/app/search/d2-known-values.ts

```diff
@@ -242,6 +242,12 @@ export const VENDORS = {
   VAULT: 1037843411,
   XUR: 2190858386,
   WAR_TABLE_UPGRADES_RISEN: 3950870173,
+  /**
+   * this has always been named "The Gate Lord's Eye" from season 8,
+   * but every season this vendor is updated with the new contents
+   * of that season's artifact
+   */
+  ARTIFACT: 2894222926,
```

Comment:
I had previously solved this by actually looking at the artifact in the inventory but hardcoding seems fine to me too

https://github.com/DestinyItemManager/DIM/blob/15b69b7e2c7c3fb206488b82a6aa21ebaea28b14/src/app/strip-sockets/strip-sockets.ts#L55-L57